common/config/rush/pnpm-lock.yaml

```diff
@@ -5747,11 +5733,11 @@ packages:
       '@itwin/core-quantity': 3.6.0 - 3.6.1
       '@itwin/ecschema-metadata': 3.6.0 - 3.6.1
     dependencies:
-      '@itwin/core-backend': link:../../core/backend
-      '@itwin/core-bentley': link:../../core/bentley
-      '@itwin/core-common': link:../../core/common
-      '@itwin/core-geometry': link:../../core/geometry
-      '@itwin/ecschema-metadata': link:../../core/ecschema-metadata
+      '@itwin/core-backend': link:..\..\core\backend
```

Comment:
@arun do OS-specific path separators surprise/concern you?

Reply:
this is new, never noticed it before. idk if we should be concerned about it yet, will look into it